resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/circuitbreaker/monitoring/endpoint/events/CircuitBreakerStreamEventsDTO.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Vijay Ram
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.resilience4j.ratpack.circuitbreaker.monitoring.endpoint.events;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerConfig;
+import io.github.resilience4j.circuitbreaker.event.CircuitBreakerEvent;
+
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CircuitBreakerStreamEventsDTO {
+
+    private CircuitBreakerEvent circuitBreakerRecentEvent;
+    private CircuitBreaker.Metrics metrics;
+    private CircuitBreaker.State currentState;
+    private float failureRateThreshold;
+    private float slowCallRateThreshold;
+
+    public CircuitBreakerStreamEventsDTO(CircuitBreakerEvent circuitBreakerEvent,
+                                         CircuitBreaker.State state,
+                                         CircuitBreaker.Metrics metrics,
+                                         CircuitBreakerConfig circuitBreakerConfig) {
+        this.circuitBreakerRecentEvent = circuitBreakerEvent;
+        this.metrics = metrics;
+        this.currentState = state;
+        this.failureRateThreshold = circuitBreakerConfig.getFailureRateThreshold();
+        this.slowCallRateThreshold = circuitBreakerConfig.getSlowCallRateThreshold();
```

Comment:
Added similar to springboot2 health details.

https://github.com/resilience4j/resilience4j/blob/b4f5249034d5bdea7ba78980b09253a4712a4bd2/resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/health/CircuitBreakersHealthIndicator.java#L59